### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by saltmild** [bitfocus/companion#2954]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I seem to have a bug where not all buttons are getting imported from a config export, repeatable across multiple machines.
Buttons will import, but after page 5 (in this instance) the button stop showing any text/images applied. The buttons do however still has the correct actions shown and button presses still work in emulator or controller. But still appear as empty buttons. 
Buttons also cant be edited or proper deleted to make changes.

Tried importing on multiple machines/ instances and the issue repeats on windows and mac, but also some instances of windows and mac have no issue and import the whole file no issue. All using same version.

<img width="1267" alt="Screenshot 2024-07-14 at 22 38 29" src="https://github.com/user-attachments/assets/c3436e39-58ee-47cb-b122-29dc71119850">

Tried both compressed and uncompressed full/single page exports, latest 3.3.1 stable, and other versions from 3.0.1 also.

### Steps To Reproduce

Import file (full / single page. compress/ed) from an export. After certain number of buttons the buttons start to apear as unpopulated empty, but still hold actions and can be triggered.

### Expected Behavior

The buttons would full import with text/ png images.

### Environment (please complete the following information)

```markdown
Tested on
- OS: Windows / Mac
- Browser: Chrome/ Arc/ Edge
- Companion Version: 3.3.1 Stable
```

### Additional context

Can send over file for testing

**Comment by Julusian** [comment]
This is a known bug https://github.com/bitfocus/companion/issues/2832 when the renderer encounters some emoji/unicode characters it crashes and buttons stop getting drawn.
It is not an easy/safe fix, so is only fixed in the 3.4 beta builds

**Comment by saltmild** [comment]
Thank you. Beta 3.4 is working great
